continuedev/continue
# Updates to running the doc server locally

## Description

### CONTRIBUTING.md
- _changed 'navigate to project root directory' to 'navigate to docs subdirectory'_

It seems like the process has changed, since you can't run npm from the root directory, there's no package.json there.  So I updated the instructions to start from the 'doc' subdir.

- _Added 'npm install' to instructions_

This is required for new installs, right?

### package.json 
- _Changed npm start script to use double quotes for Windows compatibility_
 
'nodemon' has some limitations using --exec on windows.  Double quotes are required for Windows users.

## Checklist


## Testing

- Viewed CONTRIBUTING doc in browser on my fork.
- Tested 'npm run start' in Windows 11 and MacOS Sonoma environments.

Follow the instructions in the updated doc to test:

1. Open your terminal and navigate to the `docs` subdirectory of the project.  The `docusaurus.config.js` file you'll see there is a sign you're in the right place.

2. Run the following command to install the necessary dependencies for the documentation server:

   ```bash
   npm install
   ```

3. Run the following command to start the documentation server:

   ```bash
   npm run start
   ```
